mPowerSDK/mPowerSDK/TasksAndSteps/APHScoreCalculator.m

```diff
@@ -108,4 +115,61 @@ + (NSArray*)convertPostureOrGain:(NSURL *)url
     return  gaitItems;
 }
 
+- (double)scoreFromTappingResult:(ORKTappingIntervalResult *)result
```

Comment:
I was having problems figuring out how to build the static PDScore library into the framework and in the interest of making what progress I could, figured I would put in this work-around until I had a chance to ask you for help and then on your first day back from vacation, I was out sick.